### PR TITLE
fix DNI projection into POA

### LINKED ISSUE #522: strange hour ahead persistence at Bend OR PV
Hard for me to explain this kind of behavior. Perhaps bad system metadata.

newplot(25)

[Forecast](https://dashboard.solarforecastarbiter.org/forecasts/single/5336aa66-a5ee-11ea-a536-0a580a80039b)

Report (UA org only): https://dashboard.solarforecastarbiter.org/reports/d900481a-c5eb-11ea-96c5-0a580a820020

### LINKED ISSUE #718: POA calculation fails to project DNI into POA 🤦
The Arbiter's power model fails to project the DNI into the plane of the array when computing POA irradiance:

https://github.com/SolarArbiter/solarforecastarbiter-core/blob/e02d458ba6635157fea0f62dc48dbf4bd67b5c75/solarforecastarbiter/pvmodel.py#L231-L233

With a quick skim of the function/module, you might see `aoi_modifier` and think POA is properly computed, but you'd be wrong. That modifier is simply the loss due to reflection off the glass. We're still missing the cosine projection. The line should read:

```python
aoi_loss = pvlib.iam.physical(aoi)
beam_effective = dni * np.cos(np.deg2rad(aoi)) * aoi_loss
poa_effective = beam_effective + poa_sky_diffuse + poa_ground_diffuse
```

Plot below shows the POA for a single axis tracker with and without the fix, plus POA calculated with ModelChain. I'm not sure about the remaining discrepancy with ModelChain as I thought I'd used consistent transposition and albedo values. So I still want to double check if there's another problem.

Screen Shot 2021-08-21 at 1 24 55 PM

Embarrassing oversight that's only now coming to light when experimenting with a clear sky power function and comparing to high quality power data. At least github added support for emojis in issue titles.

### PR DESCRIPTION
**Pull request by wholmgren**
- [x] Closes #718  closes #522 
  - [x] I am familiar with the [contributing guidelines](https://solarforecastarbiter-core.readthedocs.io/en/latest/contributing.html).
  - [x] Tests added.
  - ~Updates entries to [`docs/source/api.rst`](https://github.com/SolarArbiter/solarforecastarbiter-core/blob/master/docs/source/api.rst) for API changes.~
  - [x] Adds descriptions to appropriate "what's new" file in [`docs/source/whatsnew`](https://github.com/SolarArbiter/solarforecastarbiter-core/tree/master/docs/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
  - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
  - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by wholmgren** [comment]
Also closes #522 

## master 15 and 60 minute ahead persistence

Screen Shot 2021-08-23 at 1 18 29 PM

Screen Shot 2021-08-23 at 1 21 09 PM


## this PR 15 and 60 minute ahead persistence

Screen Shot 2021-08-23 at 1 18 20 PM

Screen Shot 2021-08-23 at 1 21 25 PM

**Comment by wholmgren** [comment]
@lboeman not sure why the tests keep failing on that one API test again. Maybe we need to wipe the test database again? (And ultimately write better core tests?)

**Comment by lboeman** [comment]
> @lboeman not sure why the tests keep failing on that one API test again. Maybe we need to wipe the test database again? (And ultimately write better core tests?)

I'm not sure why the data ends up in this state. I reset dev mysql and reran the tests. I think we need to move away from using the dev db for these tests.

**Review by lboeman — APPROVED** [review]
